CycloneDX/Services/ProjectAssetsFileService.cs

```diff
@@ -60,15 +61,16 @@ public HashSet<NugetPackage> GetNugetPackages(string projectFilePath, string pro
                     var runtimePackages = new HashSet<NugetPackage>();
                     foreach (var library in targetRuntime.Libraries.Where(lib => lib.Type != "project"))
                     {
+                        var targetAlias = TargetFrameworkToAlias(targetRuntime.TargetFramework);
                         var package = new NugetPackage
                         {
                             Name = library.Name,
                             Version = library.Version.ToNormalizedString(),
                             Scope = Component.ComponentScope.Required,
                             Dependencies = new Dictionary<string, string>(),
                             // get value from project.assets.json file ( x."project"."frameworks".<framework>."dependencies".<library.Name>."suppressParent") 
-                            IsDevDependency = SetIsDevDependency(library.Name, targetRuntime.Name, frameworksProperties),
```

Comment:
`targetRuntime.TargetFramework` is adding the runtime identifier, which is not expected.

Reply:
@NextFreeUsername This is the main change. `targetRuntime.TargetFramework.ToString()` instead of `targetRuntime.Name`. The name is internally constructed by combining the `TargetFramework` with `RuntimeIdentifier`.